qiskit_ibm_runtime/api/rest/cloud_backend.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""IBM Cloud Backend REST adapter."""
+
+from typing import Dict, Any
+
+from .base import RestAdapterBase
+from ..session import RetrySession
+
+
+class CloudBackend(RestAdapterBase):
+    """Rest adapter for IBM Cloud backend related endpoints."""
+
+    URL_MAP = {
+        "configuration": "/configuration",
+        "properties": "/properties",
+        "pulse_defaults": "/defaults",
+        "status": "/status",
```

Comment:
Where can I find the related API docs? https://us-east.quantum-computing.cloud.ibm.com/openapi/ doesn't list those endpoints.

Reply:
It's available only on the test env for now https://us-east.quantum-computing.test.cloud.ibm.com/openapi, will be pushed to prod with next release.